src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--1">
```

Comment:
```suggestion
    </div>
    
    <div class="stars stars--1">
```
<img width="1159" alt="Screenshot 2022-11-09 at 16 16 31" src="https://user-images.githubusercontent.com/104769168/200853628-6b962027-7ae2-4f93-863e-a4e6394b4f79.png">